frontend/src/models/Checklist.ts

```diff
@@ -1,7 +1,10 @@
+import {Item} from "./Item";
+
 export type Checklist = {
     id: string
     destination: string
     startDate: string
+    items: Item[]
 }
 
-export type NewChecklist = Omit<Checklist, "id">
\ No newline at end of file
+export type NewChecklist = Omit<Checklist, "id" | "items">
```

Comment:
Maybe you can define it so, then you don't need to Omit :
`id? : string  
destination: string
startDate: string 
items?: Item[]`